pkg/apis/monitoring/v1beta1/alertmanager_config_types.go

```diff
@@ -955,6 +958,136 @@ type MSTeamsConfig struct {
+type JiraField struct {
+	// Key of the tuple.
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Key string `json:"key"`
+	// Value of the tuple.
+	// +required
+	Value apiextensionsv1.JSON `json:"value"`
+}
+
+// Validate ensures JiraField is valid.
+func (jf *JiraField) Validate() error {
+	if jf.Value.Raw == nil {
+		return nil
+	}
+
+	var ret interface{}
+	dec := json.NewDecoder(bytes.NewBuffer(jf.Value.Raw))
+	dec.DisallowUnknownFields()
+	if err := dec.Decode(&ret); err != nil {
+		return fmt.Errorf("Value json decode: %w", err)
+	}
+	return nil
+}
+
+// JiraConfig configures notifications via Jira.
+// See https://prometheus.io/docs/alerting/latest/configuration/#Jira_config
+// It requires Alertmanager >= 0.28.0.
+type JiraConfig struct {
+	// Whether to notify about resolved alerts.
+	//
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty"`
+
+	// The Jira API URL i.e. https://company.atlassian.net/rest/api/2/
+	// The full API path must be included.
+	// If not specified, default API URL will be used.
+	//
+	// +kubebuilder:validation:Pattern:="^http(s)?://.+$"
+	// +optional
+	APIURL *string `json:"apiURL,omitempty"`
+
+	// The project key where issues are created.
+	//
+	// +kubebuilder:validation:MinLength=1
+	// +required
+	Project string `json:"project"`
+
+	// Issue summary template.
+	//
+	// +optional
+	Summary *string `json:"summary,omitempty"`
+
+	// Issue description template.
+	//
+	// +optional
+	Description *string `json:"description,omitempty"`
+
+	// Labels to be added to the issue.
+	//
+	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:items:MinLength=1
+	// +listType=set
+	// +optional
+	Labels []string `json:"labels,omitempty"`
+
+	// Priority of the issue.
+	//
+	// +optional
+	Priority *string `json:"priority,omitempty"`
+
+	// Type of the issue (e.g. Bug).
+	//
+	// +optional
+	IssueType *string `json:"issueType,omitempty"`
+
+	// Name of the workflow transition to resolve an issue.
+	// The target status must have the category "done".
+	// NOTE: The name of the transition can be localized and depends on the language setting of the service account.
+	//
+	// +optional
+	ResolveTransition *string `json:"resolveTransition,omitempty"`
+
+	// Name of the workflow transition to reopen an issue.
+	// The target status should not have the category "done".
+	// NOTE: The name of the transition can be localized and depends on the language setting of the service account.
+	//
+	// +optional
+	ReopenTransition *string `json:"reopenTransition,omitempty"`
+
+	// If reopenTransition is defined, ignore issues with that resolution.
+	//
+	// +optional
+	WontFixResolution *string `json:"wontFixResolution,omitempty"`
+
+	// If reopenTransition is defined, reopen the issue when it is not older than this value (rounded down to the nearest minute).
+	// The resolutiondate field is used to determine the age of the issue.
```

Comment:
Nit pic, but it seems a bit confusing as this is the documentation for the ReopenDuration and is pointing to resolutiondate. Could you please clarify?